pkg/filesystem/delete.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2023 The Rook Authors. All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package subvolume
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	exec "github.com/rook/kubectl-rook-ceph/pkg/exec"
+	"github.com/rook/kubectl-rook-ceph/pkg/k8sutil"
+	"github.com/rook/kubectl-rook-ceph/pkg/logging"
+)
+
+func Delete(ctx context.Context, clientsets *k8sutil.Clientsets, OperatorNamespace, CephClusterNamespace, subList, fs, svg string) {
+	subvollist := strings.Split(subList, ",")
+	for _, subvolume := range subvollist {
+		check := checkStaleSubvolume(ctx, clientsets, OperatorNamespace, CephClusterNamespace, subvolume)
+		if check {
+			exec.RunCommandInOperatorPod(ctx, clientsets, "ceph", []string{"fs", "subvolume", "rm", fs, subvolume, svg}, CephClusterNamespace, OperatorNamespace, true, false)
+			logging.Info("subvolume %q deleted", subvolume)
+		} else {
+			logging.Fatal(fmt.Errorf("subvolume %q is not stale", subvolume))
```

Comment:
```suggestion
			logging.Fatal(fmt.Errorf("subvolume %q is not stale", subvolume))
```

why this looging. Fatal, it should be `logging.Info` right?